modules/planning/tasks/deciders/lane_change_decider/lane_change_decider.cc

```diff
@@ -200,6 +208,9 @@ void LaneChangeDecider::PrioritizeChangeLane(
 
 void LaneChangeDecider::RemoveChangeLane(
     std::list<ReferenceLineInfo>* reference_line_info) const {
+  return;
```

Comment:
Please add a TODO here. The core dump will be fixed.

Reply:
Fixed. Thank you!